spec/models/group_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
+  subject do
+    user = User.create! name: 'john', email: 'john@test.com', password: '123456'
+    Group.create! name: 'sports', user: user
+  end
+
+  it 'test if transaction is invalid without a name' do
+    expect(subject).to be_valid
+    subject.name = ''
+    expect(subject).to be_invalid
+  end
+
+  it 'test if transaction is invalid without a user' do
+    expect(subject).to be_valid
+    subject.user = nil
+    expect(subject).to be_invalid
+  end
+end
```

Comment:
- [ ] Kindly, add as many tests as you can for your project.

![image](https://user-images.githubusercontent.com/55635977/154804498-0adc3927-4c53-4e68-812f-60db07be762c.png)

Reply:
More test have been added

Reply:
I have added a new test screenshot in the readme file